_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -0,0 +1,535 @@
+	}
+	a:hover,
+	a:focus {
+		text-decoration: underline;
+	}
+</style>
+<p>This is indicated in the cost of a <a href="#bcc0f155" role="doc-glossref">credit default swap</a>.</p>
+<h1>Glossary</h1>
+<p id="bcc0f155">
+	A credit default swap (CDS) is a financial derivative or contract that allows an investor to "swap" or offset his or
+	her credit risk with that of another investor.
+</p>
+```
+
+#### Passed Example 12
+
+This link, that is distinguishable by color (hue) from the other text in the same line, has a distinguishable bottom border when it is hovered or focused.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a {
+		color: blue;
+		background-color: white;
+		text-decoration: none;
+	}
+	a:hover,
+	a:focus {
+		border-style: solid;
+		border-color: red;
+		border-width: 0px;
+		border-bottom-width: 1px;
+	}
+</style>
+<p>Read about WAI on the <a href="http://w3.org/WAI">WAI webpage</a>.</p>
+```
+
+#### Passed Example 13
+
+This link, that is distinguishable by color (hue) from the other text in the same line, has a distinguishable box-shadow when it is hovered or focused.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a {
+		color: blue;
+		background-color: white;
+		text-decoration: none;
+	}
+	a:hover,
+	a:focus {
+		box-shadow: 4px 4px;
+	}
+</style>
+<p>Read about WAI on the <a href="http://w3.org/WAI">WAI webpage</a>.</p>
+```
+
+#### Passed Example 14
+
+This link, that is distinguishable by color (hue) from the other text in the same line, has a distinguishable font-style when it is hovered or focused.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a {
+		color: blue;
+		background-color: white;
+		text-decoration: none;
+	}
+	a:hover,
+	a:focus {
+		font-style: italic;
+	}
+</style>
+<p>Read about WAI on the <a href="http://w3.org/WAI">WAI webpage</a>.</p>
+```
+
+#### Passed Example 15
+
+This link, that is distinguishable by color (hue) from the other text in the same line, has an icon that appears when the link is focused or hovered. This icon indicates the element is a link.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a {
+		color: blue;
+		background-color: white;
+		text-decoration: none;
+	}
+	.hide {
+		display: none;
+	}
+	a:hover img.hide,
+	a:focus img.hide {
+		display: inline;
+	}
+</style>
+<p>
+	Read about WAI on the
+	<a href="http://w3.org/WAI">WAI webpage <img class="hide" src="/test-assets/be4d0c/icon.png" alt="" /></a>.
+</p>
+```
+
+#### Passed Example 16
+
+This link had a descendant element that is distinguishable by color (hue) from the other text in the same line and has a distinguishable bottom border when it is hovered or focused.
+
+```html
+<style>
+	a {
+		text-decoration: none;
+	}
+
+	a:hover span,
+	a:focus span {
+		border-bottom: 2px solid currentColor;
+	}
+</style>
+
+<p>
+	Read about WAI on the <a href="http://w3.org/WAI"><span>WAI webpage</span></a
+	>.
+</p>
+```
+
```

Comment:
I think we need to illustrate the interaction state. So probably a link whose distinguishing stuff changes whether it's `:link` or `:visited`. (ideally changing between style and content to make the meaning of the alternate quantifiers clear)

Reply:
Because of privacy concerns, one cannot do everything it pleases with the `:visited` pseudo-classes (e.g. cannot use `:after`, which would be nice to add content). But I'll come up with something based on the different styles that are available for change.